esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3004,6 +3005,23 @@ void WsWuHelpers::submitWsWorkunit(IEspContext& context, IConstWorkUnit* cw, con
     const char *paramXml, IArrayOf<IConstNamedValue> *variables, IArrayOf<IConstNamedValue> *debugs, IArrayOf<IConstApplicationValue> *applications)
 {
     ensureWsWorkunitAccess(context, *cw, SecAccess_Write);
+
+#ifndef _NO_LDAP
+    CLdapSecManager* secmgr = dynamic_cast<CLdapSecManager*>(context.querySecManager());
```

Comment:
If there is no security manager configured, this code will never allow a workunit to be submitted. We need skip over this checkViewPermission test if there is no security manager. Please make sure you comment the code otherwise it will look like a bug